src/Skyhop.Mail.Abstractions/Skyhop.Mail.Abstractions.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(MSBuildThisFileDirectory)\PackageDetails.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
perhaps start multi targetting net5 & netcore3.1 at the same time?

Reply:
Probably out of scope for this PR

Reply:
Nah just shovelled it in.

Reply:
Yes, but net5.0 doesn't work well with microsoft......... dependencies that are 3.1 gives users of the library issues. So those dependencies should be as conditions in csproj file